app/models/claims/cloner.rb

```diff
@@ -108,8 +108,6 @@ def clone_rejected_to_new_draft(author_id:)
     draft = duplicate
     draft.transition_clone_to_draft!(author_id: author_id)
     draft
-  rescue StandardError => e
```

Comment:
What do we lose by not throwing this error here? aside from specs failing

Reply:
Good question. This exception gets caught in `ClaimsController`. This rescue is just adding `Claims::Cloner.clone_rejected_to_new_draft failed with error` to the start of the message and, as I discovered, resetting the backtrace so that the error appears to have originated here rather than the actual error.